pkg/controller/migmigration/backup.go

```diff
@@ -281,6 +282,52 @@ func (t *Task) updateBackup(backup *velero.Backup) error {
 	return nil
 }
 
+func (t *Task) deleteBackups() error {
+	client, err := t.getSourceClient()
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+
+	labels := t.Owner.GetCorrelationLabels()
+	labels["migmigration"] = string(t.Owner.GetUID())
+	list := velero.BackupList{}
+	err = client.List(
+		context.TODO(),
+		k8sclient.MatchingLabels(labels),
+		&list)
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+
+	for _, backup := range list.Items {
+		request := buildDeleteBackupRequest(backup.Name, backup.UID)
```

Comment:
The `buildDeleteBackupRequest()` function is effectively 1 line of code and only used here.  Seems simpler to just create the request here.